apps/api/lib/api_web/controllers/events_controller.ex

```diff
@@ -0,0 +1,30 @@
+defmodule ApiWeb.EventsController do
+  @moduledoc """
+  Web interface for viewing events
+  """
+
+  use ApiWeb, :controller
+
+  alias Core.EventRepo
+
+  action_fallback(ApiWeb.ErrorsController)
+
+  def index(conn, _opts) do
+    {:ok, events} = EventRepo.events()
```

Comment:
Is it possible that `&EventRepo.events/0` return something different than "ok" tuple?

Reply:
I'm quite sure not, but adding `with` block would be a good idea (and consistent with other actions)

Reply:
It could if Mnesia is down. Then, the API would return HTTP 500.